### ISSUE TITLE
QSelect options not announced by screen reader

### ISSUE THREAD

**Issue by sgoess** [quasarframework/quasar#13576]
### What happened?

When using a screen reader (e.g. VoiceOver on Mac), the options of a select menu should be announced when receiving focus by keyboard. This worked fine in Quasar V1, but doesn't work any longer with Quasar V2. When navigating through the options, the screen reader doesn't announce anything.

![Bildschirmfoto 2022-05-31 um 15 40 14](https://user-images.githubusercontent.com/17178513/171187624-d7d624ef-7a14-4d14-8382-70547cf11d9e.png)

Tested on Mac (VoiceOver) and Windows (NVDA)

### What did you expect to happen?

In V1 (https://v1.quasar.dev/vue-components/select#overview) it works as expected: 

When navigating through the options list (by keyboard), options are announced

![Bildschirmfoto 2022-05-31 um 15 43 27](https://user-images.githubusercontent.com/17178513/171188154-6e718e3a-0569-436a-970a-0ebdf7afa0af.png)

Same behavior is demonstrated on https://www.w3.org/WAI/ARIA/apg/example-index/combobox/combobox-select-only.html

### Reproduction URL

https://codepen.io/codepenuser5678/pen/WNMzYde?editors=101

### How to reproduce?

1. Open the provided URL
2. Start screen reader (VoiceOver / NVDA)
3. Using only the keyboard, tab to the select component and press enter to open the options list
4. Using the arrow keys, navigate through the options
5. Observe scren reader output
6. Compare this behavior for Quasar V1 and V2

### Flavour

Quasar CLI with Webpack (@quasar/cli | @quasar/app-webpack)

### Areas

Components (quasar)

### Platforms/Browsers

Firefox, Chrome, Safari, Electron

**Comment by rstoenescu** [comment]
Fix will be available in Quasar 2.7.2

**Comment by pdanpdan** [comment]
Hello. I was working on some changes for a11y and I need some help with testing.

The changes are related to:
- feat(QExpansionItem): add a11y for toggle buttons; improve header slot scope quasarframework#14338
- feat(QSelect): improve a11y - aria-readonly on focusable element; remove aria-activedescendant duplicate
- feat(QChip): add aria-label and proper role for remove button/icon
- feat(QDialog): set aria-modal on dialog
- feat(QMenu): set default role of menu
- feat(QFab): add a11y for toggle button; describe aria-controls/own
- feat(QBtnDropdown): add a11y for toggle buttons; describe aria-controls/own

I have published a test documentation site at https://pdanpdan.github.io/quasar-docs/ using the changed code.
If you use screen readers and/or a11y checking tools can you please check the pages for the changed components and report if there are problems or if there are improvements?
From the docs you can open codepens with the changed code, so if you have special cases you want to test you can.

**Comment by TotallyMehis** [comment]
Hey, it looks like this wasn't fixed or was reintroduced somewhere. Tested with Quasar 2.10.0 .

Example (using NVDA): 
![explorer_ICt3pwyEGz](https://user-images.githubusercontent.com/4648654/199236981-0590c74e-566b-4a65-bbca-161652e39ba6.gif)

Reproduction: https://codepen.io/TotallyMehis/pen/LYrGyBK


Works in Quasar v1. Tested here:
https://v1.quasar.dev/vue-components/select#overview

**Comment by pdanpdan** [comment]
Can you please also test it here?
https://pdanpdan.github.io/quasar-docs/vue-components/select#example--design-overview

I'll also make a VM with windows and check, but it takes time

**Comment by TotallyMehis** [comment]
> Can you please also test it here? https://pdanpdan.github.io/quasar-docs/vue-components/select#example--design-overview

Works there.

![firefox_Pg2xIC6pFe](https://user-images.githubusercontent.com/4648654/199252632-97635976-db87-4a3d-a8d2-6b8a3e7acfee.gif)

**Comment by pdanpdan** [comment]
I found what's changed between Qv1 and Qv2 when using QSelect with options as menu.
Can you please also check if QSelect with options as dialog is read as expected on Qv1 and not read on Qv2?

**Comment by TotallyMehis** [comment]
> Can you please also check if QSelect with options as dialog is read as expected on Qv1 and not read on Qv2?

Quasar v1 works, tested with: https://pdanpdan.github.io/quasar-docs/vue-components/select#example--show-options-in-dialog
![explorer_ZfHir1fOyG](https://user-images.githubusercontent.com/4648654/199428219-0d84c113-99e2-45f1-bcef-5d5ac6403221.gif)


Quasar v2 doesn't work. tested with: https://codepen.io/TotallyMehis/pen/LYrNPzM
![explorer_8EOpEuZV7c](https://user-images.githubusercontent.com/4648654/199428367-48a1b154-01c6-4871-9075-4fa9bb73951b.gif)

**Comment by TotallyMehis** [comment]
Seems to be fixed in 2.10.2 . Thanks for the quick fix! We really appreciate it!

**Comment by pdanpdan** [comment]
Thank you for the hint with using NVDA with speech displayed as text
I could never understand what the screen reader was mumbling

**Comment by yusufkandemir** [comment]
@pdanpdan can this be closed?

**Comment by pdanpdan** [comment]
I'll close this - if the problem shows again please comment here and I'll reopen

**Comment by sgoess** [comment]
This does still not work in macOS + Chrome + VoiceOver. The latest information about this is here: https://github.com/quasarframework/quasar/issues/17078

**Comment by yusufkandemir** [comment]
@sgoess this issue is related to QSelect itself and was fixed. The issue you linked is related to q-select inside q-dialog. If it's not working outside q-dialog either, then please clarify the things. Otherwise, please keep things in the existing issue and do not create new references.
